spec/models/converters/app_converter_spec.rb

```diff
@@ -29,4 +29,26 @@
       expect(rb.keys).to match_array apps(:app1).services.map(&:name)
     end
   end
+
+  context '#to_compose_gist' do
+    fixtures :apps, :services
+
+    let(:github_client) { double('github_client', create_gist: {}) }
+    let(:create_params) do
+      {
+        description: "created by Panamax",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.